templates/homepage.php

```diff
@@ -6,6 +6,10 @@
 
 <?php ob_start(); ?>
 <h1>Le super blog de Lewis !</h1>
+<?php  if (!isset($_SESSION['user'])) echo "<a href='index.php?action=login'>Login</a>";
+    else echo '<a href="index.php?action=logout">Log out</a><div> "bonjour' . $_SESSION['name'] . '!" </div>';
+?>
+
```

Comment:
```php
<?php if (condition): ?>
    <a href="index.php?.....">my link</a>
<?php else: ?>
    <a href="">logout</a>
<?php endif; ?>

```

Reply:
you want to check: isset + not empty (you can simply use `if !empty()` as it also validates that the variable exists